src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img 
+        class="card__image card__image--img" 
+        src="images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27&quot; Retina <br> 5K Monoblock (MNED2UA/A)"
+      >
+      <h3 class="card__title">
+        APPLE A1419 iMac 27&quot; Retina<br>5K Monoblock (MNED2UA/A)
+      </h3>
+      <div class="card_productcode">
+        Product code: 195434
+      </div>
+      <div class="card__grade">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="rewiews">Reviews: 5</div>
+        
+      </div>
+      <div class="card__price">
+        <div class="card__price-title">Price:</div>
+        <div class="card__price-cost">$2,199</div>
+      </div>
+      <!-- <div class="card__button">Buy</div> -->
+      <a 
+        href="https://www.moyo.ua/ua/monoblok_apple_a1419_imac_27_retina_5k_mned2ua_a/361498.html?
+      admitad_uid=c350e2a6f141115131a9af6e1af31b57&utm_source=admitad&utm_medium=cpa&utm_campaign=442763" 
+      class="card__button" 
+      data-qa="hover"
+        >
+        Buy
+      </a>
```

Comment:
Pay attention on indends. For unkown url just set # as href
```suggestion
      <a 
        href="#" 
        class="card__button" 
        data-qa="hover"
      >
        Buy
      </a>
```